package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "precommit": "lint-staged",
-    "prepare": "lerna bootstrap --hoist",
```

Comment:
Is there any reason why `--hoist` flag is removed? After deep diving the [Lerna Hoisting doc](https://github.com/lerna/lerna/blob/master/doc/hoist.md), seems like using the `-- hoist` flag is a more optimized setting, and a suggested practice.

Reply:
Rollup seems to have problem resolve hoisted packages.  Right now this is the only way I found how it can work properly, open to suggestions.